examples/cfd/02_convection_nonlinear.ipynb

```diff
@@ -263,27 +255,25 @@
   {
    "cell_type": "code",
    "execution_count": 7,
-   "metadata": {
-    "collapsed": false
-   },
+   "metadata": {},
    "outputs": [
     {
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "CustomCompiler: compiled /tmp/devito-pshw6um6/017c17815b4ef24496c80e316d98b03ae6fc7408.c [1.15 s]\n",
+      "CustomCompiler: compiled /var/folders/qr/4hc1nw7d3hj55jbd1z7kkrvh0000gn/T/devito-8741qqqk/2f1a291c50a8ee234a1861ccb3c37b1903e971cc.c [0.48 s]\n",
       "=========================================================================================\n",
-      "Section section_1<80,101> with OI=0.00 computed in 0.000 s [0.00 GFlops/s]\n",
-      "Section section_2<80,101> with OI=0.00 computed in 0.000 s [0.00 GFlops/s]\n",
-      "Section main<80,101,101> with OI=1.73 computed in 0.002 s [11.05 GFlops/s, 0.39 GPts/s]\n",
+      "Section section_1<81,101> with OI=0.00 computed in 0.000 s [0.00 GFlops/s]\n",
+      "Section section_2<81,101> with OI=0.00 computed in 0.000 s [0.00 GFlops/s]\n",
+      "Section main<81,99,99> with OI=1.75 computed in 0.003 s [6.53 GFlops/s, 0.23 GPts/s]\n",
```

Comment:
Again. I don't understand why the number of iterations or the OI should change.

Reply:
same as before